Orange/widgets/visualize/owboxplot.py

```diff
@@ -322,6 +291,11 @@ def set_data(self, dataset):
 
         self.dataset = dataset
         if dataset:
+            sample_at = 1000
+            if len(self.dataset) > sample_at:
+                rows = np.random.choice(len(self.dataset), size=sample_at, replace=False)
+                self.sample = Table.from_table_rows(self.dataset, rows)
+                self.sample.X = np.asarray(self.sample.X)
```

Comment:
A je to dovolj? A "Y" nas ne skrbi?

Morda bi bilo bolje imeti na DaskTable neko funkcijo, ki vrne v cisto proper navadno tabelo predelano dask table.

Reply:
Verjetno nas res skrbi. Meni se je zdel cel pristop grd in sem pustil tako kot prototip, da lahko pogledamo ce smo zadovoljni s hitrostjo.